bot/cogs/cyber.py

```diff
@@ -63,6 +63,10 @@ def __init__(self, bot: Bot):
             r"^.*\bwhat\b.*\belite\b.*\bemail\b.*$",
             re.IGNORECASE
         )
+        self.game_level_regex = re.compile(
+            r"^(le?ve?l)\b.*?(\d{1,2}).*?(\d{1,2})\b.*$",
```

Comment:
At the moment (as I understand it) this regex will fire if the Level or Challenge number is out of range, which shouldn't be the case.

Reply:
Maybe... but `:level x y` has the same problem.